code/go/0chain.net/core/memorystore/connection.go

```diff
@@ -96,8 +96,24 @@ func (s *idStats) CheckExpiredIDs() {
 var pools = make(map[string]*dbpool)
 var idS *idStats
 
-func init() {
-	DefaultPool = NewPool(os.Getenv("REDIS_HOST"), 6379)
+//func init() {
```

Comment:
Looks like forgotten code, I'll remove it since it is the same as `InitDefaultPool`.